### ISSUE TITLE
Return validation loss and metrics in the training log

### ISSUE THREAD

**Issue by Jianwei-Bao** [aws/sagemaker-python-sdk#416]
Please fill out the form below.

### System Information
- **Framework (e.g. TensorFlow) / Algorithm (e.g. KMeans)**: TensorFlow
- **Framework Version**: 1.10.0
- **Python Version**: 2.7
- **CPU or GPU**: CPU
- **Python SDK Version**:
- **Are you using a custom image**: No

### Describe the problem
I want to set up a tuning job using validation loss as the objective metric. Since the objective metrics is searched from the training log, what do I need to specify to return the validation loss in the training log?  

I'm using the TensorFlow Estimator.
